recipes-domx/meta-xt-images-vgpu/recipes-core/llvm/llvm/clang37.patch

```diff
@@ -0,0 +1,3947 @@
+diff --git include/clang/AST/ASTContext.h include/clang/AST/ASTContext.h
```

Comment:
What it the purpose of this patch? I'd love to see commit message, not just plain diff